node-gui/src/main_window/main_widget/tabs/mod.rs

```diff
@@ -70,30 +70,35 @@ impl TabsWidget {
     pub fn view(&self, node_state: &NodeState) -> Element<TabsMessage> {
         let position = self.settings_tab.settings().tab_bar_position.unwrap_or_default();
 
-        let mut tabs = Tabs::new(self.active_tab, TabsMessage::TabSelected)
+        let mut tabs = Tabs::new(TabsMessage::TabSelected)
+            .icon_font(iced_aw::ICON_FONT)
             .push(
+                0,
                 self.summary_tab.tab_label(),
                 self.summary_tab.view(node_state),
             )
             .push(
+                1,
                 self.networking_tab.tab_label(),
                 self.networking_tab.view(node_state),
             )
             .push(
+                2,
                 self.settings_tab.tab_label(),
                 self.settings_tab.view(node_state),
             );
 
-        for wallet in self.wallets.iter() {
-            tabs = tabs.push(wallet.tab_label(), wallet.view(node_state))
+        for (idx, wallet) in self.wallets.iter().enumerate() {
+            tabs = tabs.push(idx + 3, wallet.tab_label(), wallet.view(node_state))
```

Comment:
Could you read the number of tabs before the loop instead of hardcoding `+3`?

Reply:
Done.